### DISCUSSION TITLE
Conda and miniconda

### DISCUSSION THREAD

**Discussion by joseADuranC** [dat-linux/community#22] [FAQ] [answered]
Can I install conda and miniconda on DAT Linux without problems?
I am reluctant to install this program but I need it to manage environments in different projects. Is there an alternative other than Conda?

**Comment by dat-linux** [comment]
Hi, they're not really compatible systems. You can, because on Linux you can do anything you want. But DAT Linux approach is to focus on a set of apps and make them easy to install (but without conda). If you want to use anaconda you need to make sure the apps don't interfere.

**Reply by joseADuranC** [reply]
Can you give me an alternative to Conda (compatible with Dat Linux) for the management of environments in my projects?

Or on the other hand, could you offer me a web link with a tutorial for an Anaconda management in Dat Linux that does not interfere with the other applications?

Excuse me if my questions are rookie, I am quite new in Linux as well as in data science. I'd appreciate it a lot.

Post data: I am very happy with Dat Linux. Greetings from Colombia.

**Reply by dat-linux** [reply, marked as answer]
What exactly do you mean by an environment? Can you give an example? From my understanding, conda can create python environments as a convenience. Is this what you are referring to? In that case, you can create as many environments as you like, there are a range of different methods: [https://dev.to/bowmanjd/python-tools-for-managing-virtual-environments-3bko](https://dev.to/bowmanjd/python-tools-for-managing-virtual-environments-3bko)

If you use Conda it will not interact with the apps that DAT Linux manages, as it is an alternative to conda (for managing apps, not "environments". Not yet anyway, unless there's an effective way of doing it that would suit most people).

In terms of Jupyter, when you create a virtual env, just create a new kernel and when you launch Jupyter it should appear in the list of kernels you can switch into:

```
source /path/to/myenv/bin/activate
pip install ipykernel
python3 -m ipykernel install --user --name=myenv
deactivate
```

To see your kernels (do this from the Jupyter terminal tab):
`jupyter kernelspec list`
To delete a kernel:
`jupyter kernelspec uninstall kernelname`
Be careful to not delete a kernel unnecessarily

For other apps, you'd need to investigate how they similarly interact with a specific env.

Let me know if this is useful and if it works I can create some help doco for others, and maybe even a tool for it.
